1-js/02-first-steps/07-operators/article.md

```diff
@@ -1,23 +1,23 @@
-# Operators
+# Operadores
 
-We know many operators from school. They are things like addition `+`, multiplication `*`, subtraction `-`, and so on.
+Nós aprendemos muitos operadores na escolas. Alguns deles são adição `+`, multiplicação `*`, subtração `-` e assim por diante.
 
-In this chapter, we'll concentrate on aspects of operators that are not covered by school arithmetic.
+Nesta capítulo, nós vamos no concentrar nos aspectos de operadores que não são cobertos pela aritmética escolar.
```

Comment:
I would suggest for _we'll concentrate on aspects of operators_, the following translation: "iremos nos concentrar em aspectos dos operadores"